gallery_dl/extractor/tiktok.py

```diff
@@ -23,13 +23,13 @@ class TiktokExtractor(Extractor):
     cookies_domain = ".tiktok.com"
 
     def avatar(self):
-        return False
+        return ""
 
     def items(self):
         videos = self.config("videos", True)
         # We assume that all of the URLs served by urls() come from the same
         # author.
-        downloaded_avatar = not self.avatar()
+        downloaded_avatar = not bool(self.avatar())
```

Comment:
Not necessary. `not` already casts everything to `bool`.
`not ""` evaluates to `True` and `not "foobar"` to `False`